snap-runtime/src/test/java/org/esa/snap/runtime/LogUtils.java

```diff
@@ -0,0 +1,115 @@
+package org.esa.snap.runtime;
+
+import java.io.ByteArrayInputStream;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Enumeration;
+import java.util.logging.*;
+
+public class LogUtils {
```

Comment:
Better name it TestLogUtils. So that it is clear that it should only be used for testing.
Why is it necessary to configure the logger in a special way.
If some test relys on default SNAP logger settings it could fail.
Tests JP2ProductReaderTest and  JP2MetadataInspectorTest work without this configuration.
If this special logger config is needed it should be revrted when the test has completed.

Reply:
From my point of view the class name should not contain the 'Test' word because it does not contain test methods.
Indeed it is used in the test classes to configure the logger.

Reply:
Then it could be named Log4TestUtils or just Log4Test.

However, there is still the issue that it changes the global state.
This is never good because this can break other tests. Maybe not very critical for the logger.

So why is this special logger configuratio necessary. What's the intention?

Reply:
The intention with the class (which configures the logger) was to enable/disable some loggers in order to display only certain message from the invoked methods.
For example: when a GPT test was running I wanted to display only the log messages from the test class and the log messages from the classes belonging to the product reader module was disabled, using a system property '-DloggerName.level=OFF'.